test/message/nexttick_throw.js

```diff
@@ -5,7 +5,9 @@ process.nextTick(function() {
   process.nextTick(function() {
     process.nextTick(function() {
       process.nextTick(function() {
+        /* eslint-disable no-undef */
```

Comment:
Does our version of ESLint support `eslint-disable-next-line`? It would make these types of changes only require one line.